src/librustc_mir/transform/const_prop.rs

```diff
@@ -103,11 +115,154 @@ impl<'tcx> MirPass<'tcx> for ConstProp {
     }
 }
 
+struct ConstPropMachine;
+
+impl<'mir, 'tcx> interpret::Machine<'mir, 'tcx> for ConstPropMachine {
+    type MemoryKinds= !;
```

Comment:
Missing space before `=`.

Reply:
@wesleywiser just a reminder for this. I hope these post-landing reviews are not just ignored?

Reply:
@RalfJung I made this change in a follow up PR (#64890) which you may want to look at as well.